docs/geedocs/docsrc/BUILD_DOCS.md

```diff
@@ -0,0 +1,175 @@
+
+    Commit your changes and create a PR.
+
+__NOTE:__
+## NOTE: When you merge your release branch in to your master with doc changes
+
+Assume you have branched to release and your release branch is release_5.3.2
+and you are making document changes that need to go in to the release_5.3.2
+branch and master.
+
+ 1) Create a branch off release_5.3.2 branch:
+
+    ```bash
+    $ git checkout -b release_5.3.2 upstream/release_5.3.2
+    $ git checkout -b changes_to_5.3.2
+    ```
+
+ 2) Make your changes as in step 1 or step 2 above. Perform step 3 to generate html.
+    This step will update earthenterprise/docs/geedocs/5.3.2
+
+3) Commit rst and 5.3.2 html changes and create a PR to push to
+   release_5.3.2 branch.
+
+4) After your changes are merged in to release_5.3.2 branch,
+   To merge doc changes to master:
+
+   a) Update master in your fork:
+      ```bash
+      $ git checkout master
+      $ git fetch upstream
+      $ git pull upstream master ( to get the latest changes from upstream)
+      $ git push origin master
+      ```
+
+   b) Create a branch off of master:
+      ```bash
+      $ git checkout master
+      $ git checkout -b merge_5.3.2_to_master
+      ```
+
+   c) Pull release_5.3.2 changes:
+
+      ```bash
+      $ git pull upstream release_5.3.2
+        From git://github.com/google/earthenterprise
+        * branch            release_5.3.2 -> FETCH_HEAD
+        Auto-merging docs/geedocs/docsrc/index.rst
+        CONFLICT (content): Merge conflict in docs/geedocs/docsrc/index.rst
+        Automatic merge failed; fix conflicts and then commit the result.
+      ```
+
+      There was a merge conflict in index.rst.
+
+      ```bash
+      <<<<<<< HEAD
+      Google Earth Enterprise Documentation Version |release|
+      =======================================================
+
+         This documentation contains legacy information that may not apply
+         to the Google Earth Enterprise Open Source version. This bundle is
+         also available in the GEE Server |release| installation package.
+      =======
+      Google Earth Enterprise Documentation Version 5.3.3
+      ===================================================
+
+         This documentation contains legacy information that may not apply
+         to the Google Earth Enterprise Open Source version. This bundle is
+         also available in the GEE Server 5.3.3 installation package.
+      >>>>>>> 21627632956e75c4255d0701127556cbd46e6b31
+      ```
+
+      Keep the change, which has |release| instead of 5.3.3. This will help
+      update the documentation based on the current opengee release.
+
+      Merges may become tricky if there are conflicts in the generated html.
+      if for example, we are bring 5.3.2 changes in to master we will be having 5.3.2
+      html changes. you can keep the change in 5.3.2 branch while resolving conflicts.
+      Resolve conflicts in rst and generate html for master which will update 5.3.3
+      html.
```

Comment:
Would we not want to just blindly accept the html generated results?  If so I'm not sure how things become tricky?

Reply:
It should be straight forward... unless some master changes accidentally got in to release_5.3.2 folder like last time. We should never marge master changes in to release_5.3.2 folder after we branched. If we need to fix something after we branched, we need to push to 5.3.2 branch first and then merge those changes in to master.